sonar_dave_plugins/sonar_dave_uuv/src/gazebo_ros_image_sonar.cpp

```diff
@@ -49,6 +49,7 @@
 #include <sensor_msgs/image_encodings.h>
 #include <cv_bridge/cv_bridge.h>
 #include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc_c.h>
```

Comment:
Why are we making changes to the image sonar?

Reply:
@bsb808 There is a compiling error. It's fixed by adding imgproc_c.h